homeassistant/components/switch/ecoplug.py

```diff
@@ -0,0 +1,85 @@
+"""
+EcoPlugs Component for aut-discovering Wion and EcoPlugs automatically.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/ecoplug/
+"""
+
+import logging
+
+from datetime import timedelta
+
+from homeassistant.const import (DEVICE_DEFAULT_NAME, EVENT_HOMEASSISTANT_STOP)
+from homeassistant.components.switch import SwitchDevice
+
+DEFAULT_INVERT_LOGIC = False
+
+DOMAIN = 'ecoplug'
```

Comment:
The platform domain is the domain of the base component, eg in this case that's `'switch' `.

Just remove this.